docs/core/diagnostics/dotnet-counters.md

```diff
@@ -29,6 +29,9 @@ There are two ways to download and install `dotnet-counters`:
   | macOS   | [x64](https://aka.ms/dotnet-counters/osx-x64) |
   | Linux   | [x64](https://aka.ms/dotnet-counters/linux-x64) \| [arm](https://aka.ms/dotnet-counters/linux-arm) \| [arm64](https://aka.ms/dotnet-counters/linux-arm64) \| [musl-x64](https://aka.ms/dotnet-counters/linux-musl-x64) \| [musl-arm64](https://aka.ms/dotnet-counters/linux-musl-arm64) |
 
+> [!NOTE]
+> To use `dotnet-counters` on an x86 app, you need a corresponding x86 version of the tool.
```

Comment:
It is not clear how to do this.  It might be nice to have more detailed instructions somewhere.  Having two versions of the tool on Windows is a bit difficult.  I was using the same tool but executing it with the x86 runtime by pointing `dotnet` at `dotnet-dump.dll`.  It is a bit awkward experience.

This also affects arm32 when analyzing cross dumps on Windows.  So you currently need a 32bit dotnet tool process to debug a 32 bit target process or 32 bit dump.

So I guess you would see a similar problem on a arm64 host running a arm32 process.  Although I can't say I have seen that problem.  So it might not exist.

Reply:
Thanks for the feedback Steve. 

AFAIK the easiest/pain-free way for the customers to do this would be to just use the single-file versions of these apps which is why I added the remark here (the most natural thing to do after reading this comment would be to just click the x86 link above).

Reply:
Its OK to leave as is, I didn't really get my bearings.  You could consider something like this ...
```suggestion
> To use `dotnet-counters` on an win-x86 app, you need a corresponding [win-x86](https://aka.ms/dotnet-counters/win-x86) direct download version of the tool.
```